### ISSUE TITLE
a11y fixes missing from latest release

### ISSUE THREAD

**Issue by lc512k** [Financial-Times/o-buttons#102]
Hi all,

The focus state changes for a11y went into 4.3.1 but aren't included in 4.4.3 (4.3.1 was released after 4.4.3)

n-ui currently pulls the latest version of o-buttons and the changes in 4.3.1 are missing from it.

Can we get a new 4.4.x release that includes everything? (I didn't want to release it myself in case there was a reason for releasing this change as a 4.3.x)

**Comment by onishiweb** [comment]
@lc512k is this fixed now with the 4.4.4 release you made?

**Comment by lc512k** [comment]
Yup, just checked, changes are now live on the front page. Thank you!
